packages/vite-plugin-cloudflare/e2e/helpers.ts

```diff
@@ -99,6 +117,16 @@ function wrap(proc: childProcess.ChildProcess): Process {
 			return closePromise.then(([exitCode]) => exitCode ?? -1);
 		},
 	};
+
+	onTestFailed(() => {
+		console.log(
+			`Wrapped process logs (${proc.spawnfile} ${proc.spawnargs.join(" ")}):`
+		);
+		console.log(wrappedProc.stdout);
+		console.log(wrappedProc.stderr);
```

Comment:
Better to use `console.error` here?

```suggestion
		console.error(wrappedProc.stderr);
```